service/src/main/java/bio/terra/pipelines/service/PipelineRunsService.java

```diff
@@ -124,18 +135,16 @@ public Map<String, Map<String, String>> preparePipelineRun(
    * <p>This SAS url along with the source file path provided by the user are used to generate an
    * azcopy command that the user can run to upload the file to the location in the pipeline
    * workspace storage container.
+   *
+   * <p>The TSPS database will auto-generate created and updated timestamps, so we do not need to
```

Comment:
not sure this comment needs to exist in general but i dont think it has anything to do with the function its commenting on as well

Reply:
nice catch, copy pasta mistake